src/plot_utils.py

```diff
@@ -79,7 +79,7 @@ def mi_hist(bdb, generator, col1, col2, num_samples=1000, bins=5):
 
 
 def heatmap(bdb, bql, vmin=None, vmax=None, row_ordering=None,
-        col_ordering=None):
+            col_ordering=None):
```

Comment:
In this case, I disagree: I find the matching indent easier to read, as pylint suggests. But it's moot.
In general, I'd rather abide by standards that we're going to encounter in the outside world, i.e. pylint's defaults, rather than adopting our own, but this isn't the moment that I feel the need to change it.
More importantly, I think pylint is good at pointing out real bugs, so I'd like to abide by it so it can help us write less buggy code.
Anyway, reverted for now.